content/tutorials/token-authentication.textile

```diff
@@ -180,7 +235,7 @@ Token requests, unlike tokens, are created and signed by your server without hav
 
 Therefore, as it is possible to issue token requests without any communication to Ably (in the form of an HTTP request), we recommend that when a client needs to authenticate, you should provide a signed token request. This approach minimises the amount of work your servers need to do by distributing the task of obtaining a token from the Ably service using the token request to your clients instead.
 
-In this step we will get the local web server ready to respond to authentication requests by issuing a token request to the requesting client. By adding the following route to your Express.js server, it is now ready to service clients wanting to authenticate with Ably.
+In this step we will get the local web server ready to respond to authentication requests by issuing a token request to the requesting client. By adding the following route to your server, it is now ready to service clients wanting to authenticate with Ably.
```

Comment:
Do you think "the following route" is suitable terminology for the PHP demo as well?  Not sure, I'd like your view on whether we should customise this text per language if you think route is not appropriate for PHP.